### PR TITLE
[TODO] brew doesn't require sudo

### PR DESCRIPTION
**Pull request by timotheecour**

### CODE REVIEW

File: readme.markdown [hunk 1]
```diff
@@ -727,7 +727,7 @@ installing your package (on macOS):
 ```
   Hint: This package requires some external dependencies.
   Hint: To install them you may be able to run:
-  Hint:   sudo brew install openssl
```
Comment (dom96) [inline readme.markdown]: This is taken straight from Nimble's output, modifying this is really not the right thing to do. Fix the output.

Comment (timotheecour) [reply readme.markdown]: i don't understand; here's what I'm seeing:

drange.nimble:
```
import distros
foreignDep "openssl"
```
nimble install
```
nimble install
  Verifying dependencies for drange@0.1.0
 Installing drange@0.1.0
    Prompt: drange@0.1.0 already exists. Overwrite? [y/N]
    Answer: y
   Success: drange installed successfully.
      Hint: This package requires some external dependencies.
      Hint: To install them you may be able to run:
      Hint:   brew install openssl
```
=> it doesn't show `sudo`

**Comment by Araq** [comment]
@timotheecour This is a distros.nim bug, I fixed it for you.

**Comment by Araq** [comment]
Well it will be gone with a new Nimble release.

**Comment by timotheecour** [comment]
looks like none of us were on the same page here, the `This is a distros.nim bug, I fixed it for you.` was in fact going the opposite direction (making brew require sudo, https://github.com/nim-lang/Nim/commit/0ecec8362a8a110603fc0a7afd4b3f74287074ef)

I'm fixing this here: https://github.com/nim-lang/Nim/pull/10525 so that msg doesn't show `sudo brew install foo`

**Pull request merged**